src/apexcharts.js

```diff
@@ -128,10 +128,9 @@ export default class ApexCharts {
     gl.comboCharts = combo.comboCharts
     gl.comboBarCount = combo.comboBarCount
 
-    if (
-      ser.length === 0 ||
-      (ser.length === 1 && ser[0].data && ser[0].data.length === 0)
-    ) {
+    const allSeriesAreEmpty = ser.every(s => s.data && s.data.length === 0)
+
+    if (ser.length === 0 || allSeriesAreEmpty) {
```

Comment:
before it was checking only for 1 series for no data,
but better to check all passed series.data for an empty array to set an appropriate "No Data" state